assets/wire/components/WirePreview.tsx

```diff
@@ -95,6 +96,27 @@ class WirePreview extends React.PureComponent<any, any> {
                             />
                         )
                     )}
+                    {allMedia == null ? null : allMedia
```

Comment:
seems like this is pretty much the same as in detail, would be good to have a component for it